### ISSUE TITLE
rpm package version doesn't match release version

### ISSUE THREAD

**Issue by smougenot** [christiangalsterer/httpbeat#23]
The release file httpbeat-4.0.0-x86_64.rpm contains a package with version 5.2.1_SNAPSHOT

```
$> rpm -qip  httpbeat-4.0.0-x86_64.rpm 
Name        : httpbeat                     Relocations: / 
Version     : 5.2.1_SNAPSHOT                    Vendor: Elastic
Release     : 1                             Build Date: dim. 19 févr. 2017 08:31:43 CET
Install Date: (not installed)               Build Host: bb0646f58027
Group       : default                       Source RPM: httpbeat-5.2.1_SNAPSHOT-1.src.rpm
Size        : 29246589                         License: ASL 2.0
Signature   : (none)
Packager    : <@bb0646f58027>
URL         : https://github.com/christiangalsterer/httpbeat
Summary     : Queries arbitrary HTTP endpoints in a regular interval and ships the result to the configured output channel.
Description :
Queries arbitrary HTTP endpoints in a regular interval and ships the result to the configured output channel.```

I couldn't find where this rpm 'dedicated' version is stored in the repo (so I've no PR to fix it)

**Comment by christiangalsterer** [comment]
This is kind of a know issue by the beats infrastructure, that for the version not the beat version is used  but the version of the used beats infrastructure. I will look into it again, if this has been maybe fixed in a newer version of the beats infrastructure.

**Comment by christiangalsterer** [comment]
This will be fixed with the next version 4.1.0 which will use Beats 5.6.2.
